### PR TITLE
Update FIPS140-3 exclusions with Windows

### PR DESCRIPTION
**Pull request by JinhangZhang**
Add failures shows up in the windows platform.

### CODE REVIEW

**Comment by AdamBrousseau** [comment]
Jenkins line endings check

**Comment by AdamBrousseau** [comment]
Jenkins copyright check

**Comment by JinhangZhang** [comment]
@keithc-ca please help to review

File: test/jdk/ProblemList-FIPS140_3_OpenJcePlus.txt [hunk 1]
```diff
@@ -20,967 +20,962 @@
...
+sun/security/ec/TestEC.java https://github.com/ibmruntimes/openj9-openjdk-jdk/issues/820 linux-ppc64le,linux-s390x,linux-x64,windows-all
+sun/security/ec/ed/EdCRLSign.java https://github.com/ibmruntimes/openj9-openjdk-jdk/issues/820 aix-all,linux-ppc64le,linux-s390x,linux-x64,windows-all
+sun/security/ec/ed/EdDSAKeyCompatibility.java https://github.com/ibmruntimes/openj9-openjdk-jdk/issues/820 aix-all,linux-ppc64le,linux-s390x,linux-x64,windows-all
+sun/security/ec/ed/EdDSAKeySize.java https://github.com/ibmruntimes/openj9-openjdk-jdk/issues/820 aix-all,linux-ppc64le,linux-s390x,linux-x64,windows-all
+sun/security/ec/ed/EdDSANegativeTest.java https://github.com/ibmruntimes/openj9-openjdk-jdk/issues/820 aix-all,linux-ppc64le,linux-s390x,linux-x64,windows-all
+sun/security/ec/ed/EdDSAParamSpec.java https://github.com/ibmruntimes/openj9-openjdk-jdk/issues/820 aix-all,linux-ppc64le,linux-s390x,linux-x64,windows-all
+sun/security/ec/ed/EdDSAReuseTest.java https://github.com/ibmruntimes/openj9-openjdk-jdk/issues/820 aix-all,linux-ppc64le,linux-s390x,linux-x64,windows-all
+sun/security/ec/ed/EdDSATest.java https://github.com
```
Comment (keithc-ca) [inline test/jdk/ProblemList-FIPS140_3_OpenJcePlus.txt:981]: The file should end with a newline.

**Review by keithc-ca — CHANGES_REQUESTED** [review]

File: test/jdk/ProblemList-FIPS140_3_OpenJcePlus.txt (hunk 1 above)
Comment (JinhangZhang) [reply test/jdk/ProblemList-FIPS140_3_OpenJcePlus.txt]: added

**Review by keithc-ca — APPROVED** [review]

**Pull request merged**